pytrickle/client.py

```diff
@@ -114,13 +121,14 @@ async def stop(self):
         # Send sentinel value to stop egress loop
         try:
             self.output_queue.put_nowait(None)
+            await self.data_queue.append(None)
         except queue.Full:
             pass
```

Comment:
Can we add one warning log line here with when the queue is full?

Reply:
That is the output queue right? Not the data queue. Happy to add that in a separate PR while also changing it to wither an `asyncio.Queue` or a `Deque`

Reply:
Ah, yes. It's a deque. Apparently this will not throw a `QueueFull` error, but can be logged once the length hits max https://docs.python.org/3.10/library/collections.html?highlight=deque#collections.deque. 

Sure, let's handle this in another PR.

I'm thinking we can add something like this or go back to a regular queue if catching full is more performant:
```
if len(self.data_queue) ==  self.data_queue.maxlen:
    print("Discarding message:", self.data_queue[0])
self.data_queue.append(new_item)
```